### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by alalawy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,47 @@
     <title>Send to Whatsapp</title>
     <link href="https://bootswatch.com/flatly/bootstrap.min.css" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```
Comment (achmadfatoni) [inline index.html]: no need to add extra bootstrap css file and jquery. index.html already have bootstrap css file

Comment (alalawy) [reply index.html]: But Your bootstrap css file not supported for navbar.
so be better if you change your bootstrap css to official bootstrap cdn.

File: index.html [hunk 2]
```diff
@@ -6,8 +6,47 @@
...
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
+   
 </head>
 <body>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (achmadfatoni) [inline index.html]: as defined in issue, create page title only `Send to whatsapp` and please remove another menu.

File: index.html (hunk 1 above)
Comment (achmadfatoni) [reply index.html]: you can remove my old bootstrap css file and also remove jquery.
i think no need jquery for now. since i will not using dropdown menu.

**Pull request merged**